src/backend/config.ts

```diff
@@ -296,6 +296,7 @@ class GlobalConfigV0 extends GlobalConfig {
       addStartMenuShortcuts: false,
       autoInstallDxvk: true,
       autoInstallVkd3d: true,
+      autoInstallDxvkNvapi: false,
```

Comment:
If this is true by default, would lead to something breaking on non Nvidia or not supported cards?

Reply:
no, but it's better to have it off by default than on, since proton does it the same way for some reason that I am not aware of